packages/core/src/utils/dateUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { DateTime } from "luxon";
```

Comment:
See my other comment on the necessity of luxon. Looking at this file, it does not seem like we need any hardcore date formatting that `Date` does not provide already.  Could we remove luxon?

Reply:
Honestly after looking at the equivalent with normal javascript date objects, I prefer if we just stick with luxon.

```ts
date.toFormat("MMM dd, yyyy");
```

Is way easier to read and understand than

```ts
function mddyyyy_to_mmmdyyyy(s) {
  var s = s.split(/\D/),
    dt = new Date(s[2], s[0] - 1, s[1]);
  return dt.toLocaleString('en-CA', {
    month: 'short',
    day: 'numeric',
    year: 'numeric'
  });
}
```
[Source](https://stackoverflow.com/questions/49025561/convert-date-string-into-mmm-d-yyyy-format-in-javascript)

Reply:
The date example is way to convoluted. Either the author did this purposfully, or it is outdated. Given a valid date, the two methods compare like this:

```ts
return date.toFormat("MMM dd, yyyy");
```
vs
```ts
const day = date.toLocaleString(undefined, {day: "2-digit"});
const month = date.toLocaleString(undefined, {month: "short"});
const year = date.getFullYear();
return `${month} ${day}, ${year}`;
```

which to me reads better anyways (I had to look up the formatting codes for the moment conversion to understand what it tried to output). 
Despite of this, these are all hidden in utility methods, so we'd only have to write them once. I think depending on a library for turning the vanilla `Date` method to a (somewhat less understandable) one-liner is overkill, and we should only use it if we see the need for it in the future.
I won't have this stop the PR though. If you want to "legge denne død", just resolve this conversation and merge if you don't agree with me.

Reply:
Fixed